arch/arm/core/cortex_m/exc_manage.c

```diff
@@ -15,14 +15,14 @@
 #include <misc/printk.h>
 void sys_exc_esf_dump(NANO_ESF *esf)
 {
-	printk("r0/a1:  %x  ", esf->a1);
-	printk("r1/a2:  %x  ", esf->a2);
-	printk("r2/a3:  %x\n", esf->a3);
-	printk("r3/a4:  %x  ", esf->a4);
-	printk("r12/ip: %x  ", esf->ip);
-	printk("r14/lr: %x\n", esf->lr);
-	printk("r15/pc: %x  ", esf->pc);
-	printk("xpsr:   %x\n", esf->xpsr);
+	printk("r0/a1:  %x  ", esf->basic.a1);
```

Comment:
very minor suggestion: use '0x%08x' so it prints nicely

Reply:
Done, thanks